packages/block-editor/src/components/block-preview/auto.js

```diff
@@ -18,37 +22,51 @@ function AutoBlockPreview( { viewportWidth, __experimentalPadding } ) {
 		{ width: containerWidth },
 	] = useResizeObserver();
 	const [
-		containtResizeListener,
+		contentResizeListener,
 		{ height: contentHeight },
 	] = useResizeObserver();
+	const styles = useSelect( ( select ) => {
+		return select( store ).getSettings().styles;
+	} );
 
 	// Initialize on render instead of module top level, to avoid circular dependency issues.
 	MemoizedBlockList = MemoizedBlockList || pure( BlockList );
 
-	const scale =
-		( containerWidth - 2 * __experimentalPadding ) / viewportWidth;
+	const scale = containerWidth / viewportWidth;
 
 	return (
-		<div
-			className="block-editor-block-preview__container editor-styles-wrapper"
-			aria-hidden
-			style={ {
-				height: contentHeight * scale + 2 * __experimentalPadding,
-			} }
-		>
+		<div className="block-editor-block-preview__container">
 			{ containerResizeListener }
 			<Disabled
+				className="block-editor-block-preview__content"
 				style={ {
 					transform: `scale(${ scale })`,
-					width: viewportWidth,
-					left: __experimentalPadding,
-					right: __experimentalPadding,
-					top: __experimentalPadding,
+					height: contentHeight * scale,
```

Comment:
There's a weird behavior that is happening now in the patterns explorer because of this component (I think). When you switch to the "buttons" then the "columns" category tab in the modal patterns explorer, you'll notice that the patterns using columns (for example the three columns with images ones) jump between a full width image into the real output (kind of like moving from small screens to bigger) when first rendering. I tried solving this issue but failure for the moment, I'm not sure yet why it's rendering first like if it wasn't scaled (small screen) and then when scaled down, it shows the right sizes...